exercises/forth/canonical-data.json

```diff
@@ -0,0 +1,219 @@
+{
+  "#": [
+    "The cases are split into multiple sections, all with the same structure.",
+    "In all cases, the `expected` key is the string representation of the resulting stack",
+    "if all numbers on it are printed (top-most value on the right), separated by a space."
+  ],
+  "parsing and numbers": {
+    "cases": [
+      {
+        "description": "empty input results in empty stack",
+        "input": [],
+        "expected": ""
+      },
+      {
+        "description": "numbers just get pushed onto the stack",
+        "input": [
+          "1 2 3 4 5"
+        ],
+        "expected": "1 2 3 4 5"
+      },
+      {
+        "description": "all non-word characters are separators",
+        "input": [
+          "1\u00002\u00133\n4\r5 6\t7"
+        ],
+        "expected": "1 2 3 4 5 6 7"
+      }
+    ]
+  },
+  "basic arithmetic": {
+    "cases": [
+      {
+        "description": "addition and subtraction",
+        "input": [
+          "1 2 + 4 -"
+        ],
+        "expected": "-1"
+      },
+      {
+        "description": "multiplication and division",
+        "input": [
+          "2 4 * 3 /"
+        ],
+        "expected": "2"
+      },
+      {
+        "description": "division by zero results in an error",
+        "input": [
+          "4 2 2 - /"
```

Comment:
why not just `"4 0 /"`?

Is there an advantage of one or the other? Should we have both? Just `4 2 2 - /`? Just `4 0 /`?